d2layouts/d2near/layout.go

```diff
@@ -106,7 +174,34 @@ func WithoutConstantNears(ctx context.Context, g *d2graph.Graph) (nears []*d2gra
 			}
 		}
 	}
-	return nears
+	return constantNearGraphs
+}
+
+func pluckOutNearObjectAndEdges(g *d2graph.Graph, obj *d2graph.Object) (descendantsObjects []*d2graph.Object, edges []*d2graph.Edge) {
```

Comment:
just `pluckObjAndEdges`. ("out" is implied). "near" is not referenced here.

Reply:
fixed